logos/tests/unit/capacity/test_host_ram_planner.py

```
@@ -0,0 +1,133 @@
+    ok, eff, deficit = p._check_host_ram_headroom_for_cold_load(
+        provider_id=1, loading_model="m", projected_host_ram_mb=50_000.0,
+    )
+    assert ok is True
+    assert deficit == 0.0
+    assert eff == 100_000.0 - p.HOST_RAM_SAFETY_MARGIN_MB
+
+
+def test_precheck_denies_when_projected_exceeds_available():
+    """The deioma scenario: 48 GiB checkpoint vs ~15 GiB available."""
+    p = _bare_planner(_snapshot_with_host_memory(15_000.0))
+    ok, eff, deficit = p._check_host_ram_headroom_for_cold_load(
```

Ruff (0.15.14)

[warning] Unpacked variable `eff` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)